runatlantis.io/.vuepress/theme/index.js

```diff
@@ -0,0 +1,6 @@
+// introduce custom home with navbar
+// https://stackoverflow.com/a/60220684
+// https://vuepress.vuejs.org/theme/inheritance.html#usage
+module.exports = {
+  extend: '@vuepress/theme-default'
+}
```

Comment:
Before setting this, CustomHome doesn't show navbar. So add this setting and mv `components/CustomHome.vue` into `theme/components/Home.vue`